docs/docs/getting-started/03-run.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Running the image
+
+This guide assumes that the AICA software image was distributed as `aica-technology/base`. If the base image has a
```

Comment:
Should we change the default name to `registry.licensing.aica.tech/base`?

Reply:
Since that is our registry URL and we will allow user login with `docker pull`, it would make sense.

But, the `base` image pulled from ECR is not runnable directly because it needs still to be packaged, right? (creation of manifest happens in the packing script). We package and tag it manually for now, so I would prefer to keep a "simple" readable name. Then we can update this once we have completed registry access and packaging on the user-side.